cel/io.go

```diff
@@ -192,40 +191,13 @@ func RefValueToValue(res ref.Val) (*exprpb.Value, error) {
 	case types.UintType:
 		return &exprpb.Value{
 			Kind: &exprpb.Value_Uint64Value{Uint64Value: res.Value().(uint64)}}, nil
-	case types.DurationType:
-		d, ok := res.Value().(time.Duration)
-		if !ok {
-			return nil, errors.New("Expected time.Duration")
-		}
-		any, err := anypb.New(dpb.New(d))
-		if err != nil {
-			return nil, err
-		}
-		return &exprpb.Value{
-			Kind: &exprpb.Value_ObjectValue{ObjectValue: any}}, nil
-	case types.TimestampType:
-		t, ok := res.Value().(time.Time)
-		if !ok {
-			return nil, errors.New("Expected time.Time")
-		}
-		any, err := anypb.New(tpb.New(t))
-		if err != nil {
-			return nil, err
-		}
-		return &exprpb.Value{
-			Kind: &exprpb.Value_ObjectValue{ObjectValue: any}}, nil
 	default:
-		// Object type
-		pb, ok := res.Value().(proto.Message)
-		if !ok {
-			return nil, errors.New("Expected proto message")
-		}
-		any, err := anypb.New(pb)
+		any, err := res.ConvertToNative(reflect.TypeOf(&anypb.Any{}))
```

Comment:
Might want to cache the `reflect.TypeOf(&anypb.Any{})` result.

Reply:
Sure. I can do that.